packages/amplify-graphql-model-transformer/rds-lambda/handler.ts

```diff
@@ -83,3 +83,10 @@ const getDBConfig = async (): DBConfig => {
 
   return config;
 };
+
+const getDBEngine = (): string => {
+  if (!process.env.engine) {
+    throw Error('Missing database engine configuration');
+  }
+  return process.env.engine;
```

Comment:
I thought we concluded on not doing that for now and stick to the current design of reading engine from the schema input. In case of CDK construct, the `bindingType` in `modelDataSourceBinding` input will be used to deduce the engine.

Reply:
Sorry for the confusing comment: I'm referring to how we store the engine in the Lambda's environment variable. You are storing it as a string literal, rather than an SSM parameter path, which is what I recall us discussing.

Reply:
Engine doesn't disclose any sensitive DB information and was previously reviewed by AppSec to be allowed to exist in the schema. Also it can be deduced based on the modelToDatasource mapping wherever needed and not needed to be persisted and fetched from SSM.
Discussed with Chris and will sync with you on this at a later time and refactor in separate PR if needed.

Reply:
Sounds good. My concern isn't about security (Lambda Env vars are encrypted, so they're already secure) but rather about access patterns: we now have 5 parameters that we retrieve from SSM, but this special snowflake that we grab directly from the env var. I think it's fine for now; we can talk about overall storage later--we might even consider a single stringified JSON param rather than individual params, but that's a conversation for another day.